confignode/src/main/java/org/apache/iotdb/confignode/persistence/RegionInfoPersistence.java

```diff
@@ -156,8 +160,13 @@ public StorageGroupSchemaDataSet getStorageGroupSchema() {
                                   .getEndPoint());
                         }
                       });
-              schemaRegionReplicaSet.setSchemaRegionId(entity.getKey());
-              schemaRegionReplicaSet.setEndPointList(endPoints);
+              schemaRegionReplicaSet.setId(
+                  new ConsensusGroupId(GroupType.SchemaRegion, entity.getKey()));
+              // TODO: (xingtanzjr) We cannot get the dataNodeId here, use 0 as the placeholder
```

Comment:
@CRZbulabula @wangchao316 , we cannot get the DataNodeId here, so I use `0` as a temporary placeholder... Please take a look

Reply:
That's OK, I'll fix it in next PR~